laa_court_data_api_app/models/hearing/external/court_centre.py

```diff
@@ -0,0 +1,19 @@
+from uuid import UUID
+
+from pydantic import BaseModel
+
+from laa_court_data_api_app.models.hearing.external.address import Address
+
+
+class CourtCentre(BaseModel):
+    id: UUID | None
+    name: str | None
+    welsh_name: str | None
+    room_id: str | None
+    room_name: str | None
+    welsh_room_name: str | None
+    welsh_court_centre: str | None
+    short_oucode: str | None
+    oucode_l2_code: str | None
```

Comment:
I don't see `short_oucode` and `oucode_l2_code` in CDA

Reply:
https://github.com/ministryofjustice/laa-court-data-adaptor/blob/main/app/models/hmcts_common_platform/court_centre.rb